release_notes/ocp-4-7-release-notes.adoc

```diff
@@ -125,6 +125,11 @@ For more information, see xref:../authentication/managing-oauth-access-tokens.ad
 
 You can now remove or rotate the GCP admin-level credential that the xref:../operators/operator-reference.adoc#cloud-credential-operator_red-hat-operators[Cloud Credential Operator] uses in Mint mode. This option requires the presence of the admin-level credential during installation, but the credential is not stored in the cluster permanently and does not need to be long-lived.
 
+[id="ocp-4-7-compliance-cis-benchmark"]
+==== CIS Kubernetes Benchmark profile for Compliance Operator
+
+You can now use the Compliance Operator to perform Center for Internet Security (CIS) Kubernetes benchmark checks. CIS profiles for {product-title} are based on the CIS Kubernetes checks.
```

Comment:
Should "benchmark" be capitalized here as it is in the title?

Reply:
I think both should be lowercase. Great catch. Thanks!

Reply:
Actually, after more discussion with @bobfuru I'm choosing to align with I[BM Cloud docs](https://cloud.ibm.com/docs/containers?topic=containers-cis-benchmark) usage, which treats CIS Kubernetes Benchmark as a proper noun.